src/identity-user-interfaces.ts

```diff
@@ -53,6 +81,19 @@ export interface IUserAttributeChangeEvent extends BaseEvent {
     userAttributeChanges: ISDKUserAttributeChangeData;
 }
 
+// https://go.mparticle.com/work/SQDSDKS-6460
+export interface IIdentityCallback extends IdentityCallback {
+    (result: IdentityResult): void;
+}
+
```

Comment:
`IIdentityCallback` is meant to override `IdentityCallback` so that we can eventually patch `IdentityResultBody` which is missing `mpid`. I think I forgot to swap out  `IdentityResult` with `IIdentityResult`, so that should be good now.